### PR TITLE
Fix EAD count() warning, refs #13462 

### PR DESCRIPTION
**Pull request by djjuhasz**

### CODE REVIEW

File: ead.xml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+          <unittitle encodinganalog="3.1.2">Publications</unittitle>
+          <unitid encodinganalog="3.1.1" countrycode="CA" repositorycode="ON00362">S714-8</unitid>
+        </did>
+        <odd type="publicationStatus">
+          <p>published</p>
+        </odd>
+      </c>
+    </dsc>
+  </archdesc>
+</ead>
```
Comment (jraddaoui) [inline ead.xml]: Is this XML included intentionally?

**Review by jraddaoui — CHANGES_REQUESTED** [review]
Thanks @djjuhasz,

I think you added a test XML unintentionally. Also, do we have a ticket number to reference?

**Comment by jraddaoui** [comment]
I think we could use https://projects.artefactual.com/issues/13462 as reference.

File: ead.xml (hunk 1 above)
Comment (djjuhasz) [reply ead.xml]: Oops, my bad.  I was rushing to commit this last night. :(

**Comment by djjuhasz** [comment]
> I think we could use https://projects.artefactual.com/issues/13462 as reference.

I'll include the ref to 13462 in the merge commit message, and make sure the total subject line is under 52 chars.

**Review by jraddaoui — APPROVED** [review]
Thanks @djjuhasz!

50 chars, please :P ... https://chris.beams.io/posts/git-commit/#limit-50

**Comment by djjuhasz** [comment]
> 50 chars, please :P ... https://chris.beams.io/posts/git-commit/#limit-50

Boy you are strict. 😜 50 chars it is. 😆

**Pull request merged**